src/framework/textInput/TextInput.spec.js

```diff
@@ -64,5 +64,16 @@ describe('TextInput', () => {
         }
       );
     });
+
+    describe('readOnly', () => {
+      it('when input is readONly then readOnly attribute is applied', () => {
```

Comment:
`readONly` typo. And what about the `it` description rather to be `adds the readOnly attribute`?